core/templates/dev/head/components/state-editor/state-responses-editor/state-responses.directive.ts

```diff
@@ -236,6 +252,17 @@ oppia.directive('stateResponses', [
             return interactionId && INTERACTION_SPECS[interactionId].is_linear;
           };
 
+          $scope.isCurrentInteractionTrivial = function() {
+            var interactionId = $scope.getCurrentInteractionId();
+            if (
+              constants.INTERACTION_IDS_WITHOUT_ANSWER_DETAILS.indexOf(
```

Comment:
We use to set all the key-values present on contant.js into oppia.constant, so you can access this value similar to `INTERACTION_SPECS` (See line 93).

Reply:
Done